File: StudentRegistrationForm/Scripts/Scripts/StudentSummary.js
window.onload = function () {
    GetStudentSummary().then((response) => {
        console.log(response.StudentId);
        var table = document.getElementById('studentInfoTable');

        var tr = document.createElement('tr');

        var td = document.createElement('td');
        td.innerHTML = response.StudentId;
        tr.appendChild(td);

        var td = document.createElement('td');
        td.innerHTML = response.FirstName;
        tr.appendChild(td);

        var td = document.createElement('td');
        td.innerHTML = response.Surname;
        tr.appendChild(td);

        var td = document.createElement('td');
        td.innerHTML = response.Address;
        tr.appendChild(td);

        var td = document.createElement('td');
        td.innerHTML = response.PhoneNumber;
        tr.appendChild(td);

        var td = document.createElement('td');
        td.innerHTML = response.EmailAddress;
        tr.appendChild(td);

        var td = document.createElement('td');
        td.innerHTML = response.CustomDate;
        tr.appendChild(td);

        var td = document.createElement('td');
        td.innerHTML = response.GuardianName;
        tr.appendChild(td);

        var td = document.createElement('td');
        td.innerHTML = response.NationalIdentityNumber;
        tr.appendChild(td);

        var td = document.createElement('td');
        td.innerHTML = response.UserId;
        tr.appendChild(td);

        var td = document.createElement('td');
        td.innerHTML = response.SubjectTaken;
        tr.appendChild(td);

        var td = document.createElement('td');
        td.innerHTML = response.TotalMark;
        tr.appendChild(td);

        var td = document.createElement('td');
        td.innerHTML = response.StatusName;
        tr.appendChild(td);

        table.appendChild(tr);
        
    })
}

function GetStudentSummary() {
    return fetch("/StudentSummary/GetStudentSummary", {
        method: 'POST',
        headers: {
            'Content-Type': 'application/json'
        },
    })
        .then(response => { return response.json(); })
        .catch((error) => console.log(error))
}

File: StudentRegistrationForm.DAL/ViewModel/StudentSummary.cs
using Newtonsoft.Json;
using Repository.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Repository.ViewModel
{
    public class StudentSummary
    {
        public int StudentId { get; set; }
        public string FirstName { get; set; }
        public string Surname { get; set; }
        public string Address { get; set; }
        public string PhoneNumber { get; set; }
        public string EmailAddress { get; set; }
        public DateTime DateOfBirth { get; set; }

        [JsonProperty("date")]
        public string CustomDate
        {
            get { return DateOfBirth.ToString("MM/dd/yyyy"); }
            set { DateOfBirth = DateTime.ParseExact(value, "MM/dd/yyyy", null); }
        }
        public string GuardianName { get; set; }
        public string NationalIdentityNumber { get; set; }
        public int UserId { get; set; }
        public string SubjectTaken { get; set; }
        public int TotalMark { get; set; }
        public Status StatusId { get; set; }
        public string StatusName
        {
            get { return StatusId.ToString(); }
            set { StatusName = StatusId.ToString(); }
        }
    }
}
